llvm_util/llvm2alive.cpp

```diff
@@ -1570,8 +1570,11 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
         attrs.set(ParamAttrs::NonNull);
         break;
 
-      case llvm::Attribute::NoCapture:
-        attrs.set(ParamAttrs::NoCapture);
+      case llvm::Attribute::Captures:
+        if (capturesNothing(llvmattr.getCaptureInfo().getOtherComponents()))
```

Comment:
This should be just `capturesNothin(llvmattr.getCaptureInfo())`. Otherwise you're saying a return-only capture is nocapture as well.

Reply:
fixed, thank you!